MSSQL/OMOPLoader.sql

```diff
@@ -1613,43 +1613,93 @@ IF  EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'[dbo].[build
 DROP PROCEDURE [dbo].[build_modifiers]
 GO
 
-create procedure build_modifiers (@domain varchar(20)) as
-
+create procedure [dbo].[build_modifiers] (@domain varchar(20), @force_update int= 0) as

```

Comment:
Does this force it to be on dbo? Probably not a good idea in case people want to put the stored procs in another schema.

Reply:
Removed schema reference